collectd/files/plugin/contrail_apis.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/python
```

Comment:
better to name this file collectd_contrail_api.py to avoid name clashing with 3rd-party Python modules.